app/metrics.yaml

```diff
@@ -5892,6 +5910,25 @@ pocket:
     metadata:
       tags:
         - PocketIntegration
+  spoc_shown:
+    type: text
+    description: |
+      Shim data of the shown Pocket sponsored.
+      For this to be recorded the story should be >50% visible.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/27549
+    data_reviews:
+      - ?????
+    data_sensitivity:
+      - web_activity
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: 119
```

Comment:
Should this expire at all? Did you check with Pocket? I suspect this doesn't need to expire.